bootstrap.py

```diff
@@ -88,21 +88,27 @@ def choose_color_theme(window):
 
 '''})
     view.set_syntax_file('Packages/MarkdownEditing/Markdown.tmLanguage')
-    default_mde_scheme = sublime.load_settings('Markdown.sublime-settings').get('color_scheme') or 'Packages/MarkdownEditing/MarkdownEditor.tmTheme'
+    default_mde_scheme = sublime.load_settings('Markdown.sublime-settings').get('color_scheme') or 'Packages/MarkdownEditing/MarkdownEditor.sublime-color-scheme'
     print(default_mde_scheme)
     view.settings().set('color_scheme', default_mde_scheme)
     view.set_read_only(True)
     view.set_scratch(True)
 
     global_scheme = sublime.load_settings('Preferences.sublime-settings').get('color_scheme')
-    themes = ['Packages/MarkdownEditing/MarkdownEditor.tmTheme',
-              'Packages/MarkdownEditing/MarkdownEditor-Focus.tmTheme',
-              'Packages/MarkdownEditing/MarkdownEditor-Yellow.tmTheme',
-              'Packages/MarkdownEditing/MarkdownEditor-Dark.tmTheme',
-              'Packages/MarkdownEditing/MarkdownEditor-ArcDark.tmTheme',
+    themes = ['Packages/MarkdownEditing/MarkdownEditor.sublime-color-scheme',
```

Comment:
Paths are no longer needed when using `sublime-color-scheme`

Reply:
New color-scheme format also requires to resolve https://github.com/SublimeText-Markdown/MarkdownEditing/issues/602